pyensembl/biotypes.py

```diff
@@ -234,6 +235,10 @@
     'TR_V_pseudogene',
     'unitary_pseudogene',
     'unprocessed_pseudogene',
+    # to be experimentally confirmed
+    'TEC',
```

Comment:
Haha. I thought you were saying the inclusion of "TEC" needed to be experimentally confirmed. I just realized TEC *means* that.